### PR TITLE
[v6r22] New service for Pilots

### PR DESCRIPTION
**Pull request by fstagni**
This PR creates a new service (Pilots) for interacting with PilotAgentsDB. The service is created starting from WMSAdministrator service. The service is a result of a few discussions, e.g. in https://github.com/DIRACGrid/DIRAC/issues/4014 and https://github.com/DIRACGrid/DIRAC/pull/4019#discussion_r272588901

BEGINRELEASENOTES

*WMS
NEW: PilotsHandler service (to interact with PilotAgentsDB)

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
BTW this would need a PR also for the web.

File: WorkloadManagementSystem/Client/PilotsClient.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    Sets URL for Pilots handler
+
+    :param self: self reference
+    :param url: url of the PilotsHandler
+    :param kwargs: forwarded to the Base Client class
+    """
+
+    super(PilotsClient, self).__init__(**kwargs)
+
+    if not url:
```
Comment (chaen) [inline WorkloadManagementSystem/Client/PilotsClient.py]: Wouldn't it be better, as a general good practice, to define the default url as a class attribute ?

File: WorkloadManagementSystem/Service/WMSAdministratorHandler.py [hunk 2]
```diff
@@ -194,105 +162,6 @@ def export_getSiteMaskSummary(cls):
 
     return S_OK(siteDict)
 
-  ##############################################################################
-  types_getCurrentPilotCounters = [dict]
-
-  @classmethod
-  def export_getCurrentPilotCounters(cls, attrDict={}):
```
Comment (chaen) [inline WorkloadManagementSystem/Service/WMSAdministratorHandler.py]: Maybe I am mistaken, but I guess that for backward compatibility, you can't just remove all these methods at the same time you introduce the new service ?

**Comment by chaen** [comment]
Also it is maybe a bit violent for a patch no ?

**Comment by fstagni** [comment]
I agree with everything, I will now put WIP.

File: WorkloadManagementSystem/Client/PilotsClient.py (hunk 1 above)
Comment (fstagni) [reply WorkloadManagementSystem/Client/PilotsClient.py]: Maybe yes, but this would not be for this PR.

**Comment by atsareg** [comment]
The name of the service as it is coded now is Pilots. PilotsHandler is just adding a prefix. To follow the style used before I would better call it PilotManager and thus PilotManagerHandler class and PilotManagerClient class

**Pull request merged**